tests/shell/common.sh

```diff
@@ -0,0 +1,13 @@
+git-must-track () {
+  # $1: Path / filename
+  # Error if the file has never been added:
+  git ls-files --error-unmatch $1
+}
```

Comment:
Accepts only one argument?

Reply:
Yep, I don't overengineer for something I don't need yet :)